aggregator/run_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Start aggregator itself and store its PID into variable
+/home/slabimic/Nemea/Nemea-Modules/aggregator/agg -i u:input_data,u:aggr_data -k SRC_IP -k DST_IP -s BYTES -s PACKETS -t a:60 & 
```

Comment:
This is not possible and must be fixed. There is no way `/home/slabimic/...` could work in practice.